CHANGELOG.md

```diff
@@ -5,6 +5,36 @@ file is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and
 this project adheres to [Semantic
 Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Changed
+
+- Configuration options required for establishing a connection with the central
+  management server must be specified directly via command-line interface.
+  The following options and arguments are required:
+  - `--cert`: Specifies the certificate file for the current module.
+  - `--key`: Specifies the private key file for the certificate.
+  - `--ca-certs`: Specifies the CA certificate files. Multiple CA certificates
+    can be provided by using this option multiple times.
+  - The manager server's name and address must be provided as a command-line
+    argument in the format `<server_name>@<server_ip>:<server_port>`.
+- The configuration file, previously provided as a positional command-line
+  argument, must now be specified using the `-c` option. This change allows for
+  additional command-line arguments to be used alongside the configuration file.
+  However, providing a local configuration file is optional.
+- If log_dir is not specified in the configuration file or the log file cannot
+  be created, logs will be printed to stdout.
+- Default last_timestamp_data file: /usr/local/aice/conf/time_data.json
+
+### Removed
+
+- Removed cert, key, ca_certs, review_name and review_rpc_srv_addr from the
+  configuration file; now mandatory via CLI.
+- Removed OS-specific configuration directory.
+  - /home/alice/.config/barapp (Linux)
+  - /Users/Alice/Library/Application Support/com.Foo-Corp.Bar-App (macOS)
+  - C:\Users\Alice\AppData\Roaming\Foo Corp\Bar App\config (Windows)
```

Comment:
I think it would be helpful to use the actual path that was removed here: [link](https://github.com/aicers/crusher/pull/114/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L53-L56) instead of referencing Alice.

Reply:
@danbi2990, resolving conversation should be done by reviewers, which is one of our rules written in Notion.